packages/e2e-tests/helpers/timeline.ts

```diff
@@ -92,3 +92,8 @@ export async function setFocusRangeStartTime(page: Page, timeString: string): Pr
   await clearTimeInput(page, input);
   await input.fill(timeString);
 }
+
+export async function seekToPreviousScreenshot(page: Page) {
```

Comment:
Why is this called "previous screenshot" ?

Reply:
It presses the left arrow key on the timeline, which seeks to the execution point of the last screenshot before the current execution point.